src/javascript/app_2/App/Components/Elements/toggle_button.jsx

```diff
@@ -0,0 +1,20 @@
+import classNames from 'classnames';
+import React      from 'react';
+import PropTypes  from 'prop-types';
+
+const ToggleButton = ({ style, bool }) => {
```

Comment:
@zakibinary 
IMHO, it's better to change the `bool` to `toggled`. it conveys more the functionality of this prop.